src/ResourceManager/Compute/Commands.Compute/help/New-AzureRmVM.md

```diff
@@ -134,6 +134,40 @@ This script can be used for automatic provisioning because it uses the local vir
 This script assumes that you are already logged into your Azure account.
 You can confirm your login status by using the **Get-AzureSubscription** cmdlet.
 
+### Example 3: Create a VM from a marketplace image without a Public IP
+```
+$VMLocalAdminUser = "LocalAdminUser"
+$VMLocalAdminSecurePassword = ConvertTo-SecureString "Password" -AsPlainText -Force
```

Comment:
Can you change the password here to something that would need to be changed, e.g., `<password>`?